### PR TITLE
This code very fast

### PR DESCRIPTION
**Pull request by xiaozhuai**

### CODE REVIEW

**Review by rafaelreinert — APPROVED** [review]

File: src/main.cpp [hunk 1]
```diff
@@ -1,18 +1,4 @@
 #include <iostream>
```
Comment (LauraRozier) [inline src/main.cpp:1]: I doubt you need this, though.

File: src/main.cpp [hunk 2]
```diff
@@ -1,18 +1,4 @@
...
 #include "headers/siniragi.h"
 using namespace std;
```
Comment (LauraRozier) [inline src/main.cpp:3]: std? Can be removed, too.
Don't want those scary deceases in our code. :(

File: src/main.cpp [hunk 3]
```diff
@@ -1,18 +1,4 @@
...
-    deltaAgi.araKatman[0].agirliklariEkrandaGoster();
-    deltaAgi.araKatman[1].agirliklariEkrandaGoster();
-    deltaAgi.cikisKatmani.agirliklariEkrandaGoster();
-    deltaAgi.Sinif[0].beklenenDeger = 1;
-    deltaAgi.Sinif[1].beklenenDeger = -1;
-
-
-    return 0;
-}
+int main(){SinirAgi deltaAgi;deltaAgi.araKatman[0].agirliklariEkrandaGoster();deltaAgi.araKatman[1].agirliklariEkrandaGoster();deltaAgi.cikisKatmani.agirliklariEkrandaGoster();deltaAgi.Sinif[0].beklenenDeger = 1;deltaAgi.Sinif[1].beklenenDeger = -1;return 0;}
```
Comment (LauraRozier) [inline src/main.cpp:4]: Please remove the spaces, too.

**Review by LauraRozier — CHANGES_REQUESTED** [review]
Seems legit to me. 👍
Just fix the comments. :-)

**Pull request closed without merging**